### PR TITLE
Binance rest api

### PR DESCRIPTION
**Pull request by LittleberryPi**
😊

### CODE REVIEW

File: binance/REST_test.go [hunk 1]
```diff
@@ -171,14 +164,90 @@ func TestRestAPI_newOrder(t *testing.T) {
...
+	c := NewREST(APIKey, secretKey)
+	ctx := context.Background()
+	orders := AllOrdersParams{
+		Symbol:  "ARKBTC",
+		OrderId: 1,
+	}
+
+	resp, err := c.AllOrders(ctx, orders)
+	assert.NoError(t, err)
+	assert.NotNil(t, resp)
```
Comment (KaiserKarel) [inline binance/REST_test.go]: this needs to test for notemtpy, but also if allorders actually returns previously sent orders

File: binance/cancel_order.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		q.Add("origClientOrderId", params.OrigClientOrderId)
+	}
+
+	// Set symbol, recvWindow and timestamp
+	q.Add("symbol", params.Symbol)
+	if params.NewClientOrderId != "" { //Optional parameter
+		q.Add("newClientOrderId", params.NewClientOrderId)
+	}
+	if params.RecvWindow != 0 { //Optional parameter
+		recvWindowMilliseconds := params.RecvWindow.Seconds() * 1000000000 // multiply for millisecond
```
Comment (KaiserKarel) [inline binance/cancel_order.go]: this is just way too much

Comment (KaiserKarel) [reply binance/cancel_order.go]: also add a comment describing the behaviour of binance regarding timestamps

**Comment by LittleberryPi** [comment]
lgtm :)

**Pull request merged**